### DISCUSSION TITLE
역량 리포트 스쿼드 아이디어 살 붙이기

### DISCUSSION THREAD

**Discussion by wonsss** [woowacourse/prolog#913] [Ideas]
# 할 일
- 이니셔티브 후보 주제에 자신의 생각와 시나리오 등 기능을 상세할 수 있는 아이디어를 코멘트로 남겨 주세요.
- **7월 6일 오후 1시까지 모든 멤버가 코멘트 남기기**
- 7월 6일 회의에서는 코멘트 달린 내용으로 의견 합치기

# 이니셔티브 후보
## [아이디어] 리포트 작성 편의
- 학습 로그와 연동을 해서 학습 로그에 있는 태그가 자동으로 리포트에 등록
    - 보통 노션에 정리하고, 레벨 인터뷰에는 키워드를 작성
    - 태그를 이용한 카테고리 매칭은 어떻게 하면 좋을까?
- PR에서 키워드 자동 추출
    - 레벨 인터뷰 준비할 때 리뷰를 보면서 학습해야 할 내용과 키워드 정리
## [아이디어] 구현 기능
- 레벨 인터뷰 참여 & 리포트 제출
    - 레벨 인터뷰 때마다 제출하게 함으로써 리포트 사용률을 끌어 올린다
    - 다른 사람의 리포트도 볼 수 있음
- 조 매칭
- 피드백 작성
- 인터뷰 논의
    - 슬랙 채널을 따로 만들어야 하는 불편함 해결
## [아이디어] 역량 관련
- 역량과 리포트가 연결되어 있음
    - 리포트에서는 역량을 가져와서 그래프를 만드는 기능이 있음
    - 레벨로그 작성할 때는 보통 글로 작성
    - 그렇다면, 역량 그래프가 레벨 로그에 반드시 필요한 것인가?
    - 레포트는 레벨 로그만을 위해서 사용되는 것일까? 역량 그래프가 레포트에 반드시 필요할까?
- 역량과 키워드가 연관되는 것 같음
    - 키워드에 대한 질문을 적어놓고, 답을 쓰면 역량 점수가 쌓이는 느낌
- 역량 점수
    - 키워드에 대해 답변했을 때 인터뷰어가 점수 올리기
    - 키워드에 답변 글을 작성했을 때 자동으로 점수 올리기
    - 레벨 인터뷰 점수를 객관적으로 볼 수 있다면 좋겠다
        - 레벨 인터뷰 점수를 개인 피드백으로 받아들일 수 있음
        - 본인만 볼 수 있도록 하기
        - 점수를 다른 사람이 평가한다면 객관화가 어렵지 않을까?
        - 스스로 본인이 아쉬웠던 점을 평가하는 것은 어떨까?
- 역량이 무엇일까?
    - 다른 사람이 평가하는 점수가 내 역량이라고 할 수 있을까?
    - 역량, 리포트 기능은 학습에 대해 피드백을 받고 잘 학습하기 위한 목적
    - 레벨 인터뷰도 평가가 아니라 피드백을 받기 위함
- 역량 커스터마이징이 오히려 쓰기 어렵게 만드는 허들이 되는 것 같다
    - 커스터마이징이 가능하니까 오히려 무엇을 써야 할지 알기 어려움
    - 사용자마다 달라질 수 있다는 것
        -> 레벨 인터뷰에서 모아보거나 필터링하는 과정에서는 독이 될 수 있다고 생각
    - 프론트엔드 / 백엔드만 선택하고, 그 안에서 키워드는 수정할 수 없게 만들어보면 어떨까?
- 역량 탭에 있는 내용 중 잘 다루지 않는 것도 많음
    - 우테코에서 자주 다루는 키워드만 넣으면 어떨까?

**Comment by JUDONGHYEOK** [comment]
# 이니셔티브(동키콩 🦧)
## 리포트와 학습로그 연계
### 배경
- 역량을 위해 다시 리포트를 작성하는 수고로움을 덜기 위함

### 기능 상세
- 학습로그의 태그를 상세 역량으로 제한하거나 역량 탭 중 하나를 선택하여 해당 키워드가 추가되는 방식
- 해당 글을 업로드하면 역량 점수가 올라가는 형식

### 우려되는 점
- 자기 자신의 역량을 되돌아보는 메타인지의 의도와 맞는지?
- 리포트 사용비율을 올릴 수 있는 효과를 얻을 수 있는가?


## 리포트 질문 답변 형식으로 변경
### 배경
- 리포트를 올린다고 해서 역량에 대한 검증이 되는지에 대한 의문

### 기능 상세
- 리포트를 상세 역량의 탭을 기준으로 질문을 업로드 함
- 질문에 대한 답변을 서술함으로써 해당 역량의 점수가 올라감

### 우려되는 점
- 자신이 아는 질문을 위주로 업로드한다면 역량의 의미가 있는가?
